test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -362,6 +362,12 @@ Expect.describe('IDPDiscovery', function() {
       });
     });
     itp('does not show a beacon if features.securityImage is false', function() {
+
+      // BaseLoginRouter will render twice if language bundles are not loaded:
+      // https://github.com/okta/okta-signin-widget/blob/master/src/util/BaseLoginRouter.js#L202
+      // We are not testing i18n, so we can mock language bundles as loaded
+      Util.mockBundles();
```

Comment:
Hmm.. these should be loaded by default given our language code should be `en`. Is a different locale being used in this test?

Reply:
I don't think this test does any setup for i18n. But the double render was causing a race condition on the beacon. I guess the race existed but was resolving consistently with previous jquery version.

Reply:
Can we look into this further? The code block you've linked to in the comment shouldn't be executed, as we're not testing a different locale.

Perhaps something else is happening that may be indicating a bigger problem?

Reply:
the problem was a race condition. while the bundle is loading it wants to put a "loading" beacon on. But are test is trying to make assertions about the beacon. The loading logic is just in the way.